srcs_mandatory/builtin_func.c

```diff
@@ -0,0 +1,116 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   builtin_func.c                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: choiejae <choiejae@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/02/01 18:37:07 by ejachoi           #+#    #+#             */
+/*   Updated: 2023/02/06 21:49:06 by choiejae         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
+extern int g_exit_code;
+
+int	builtin_pwd(void)
+{
+	char	*buf;
+
+	buf = ft_calloc(1, MAXPATHLEN);
+	if (getcwd(buf, MAXPATHLEN) == NULL)
+		ft_printf("fail: getcwd()\n");
```

Comment:
표준출력 -> 표준에러 수정 제안합니다.

```suggestion
		ft_fprintf(2, "fail: getcwd()\n");
```